machine_learning/astar.py

```diff
@@ -137,14 +137,14 @@ def astar(world, start, goal):
 
 if __name__ == "__main__":
     world = Gridworld()
-    #   stat position and Goal
+    #   Stat position and goal.
```

Comment:
```suggestion
    #   Stat position and goal
```
You dont really need punctuation in a comment

Reply:
Got it. But the comments were inconsistent. Lower & Upper case words, either it should all be lower cased or proper english sentences.